### ISSUE TITLE
Bulletproof fontface rule too strict?

### ISSUE THREAD

**Issue by Rudloff** [CSSLint/csslint#524]
I get the bulletproof warning even if I only have one font source:

```
@font-face {
    font-family: "Quicksand";
    src: url('Quicksand-Regular.ttf');
}
```

**Comment by arlm** [comment]
Should it trigger the rule if the first element is a local? It is right now. here is the sample code:

```css
@font-face {
  font-family: 'Ubuntu';
  font-style: normal;
  font-weight: 300;
  src: url('../../fonts/ubuntu-v9-latin-300.eot'); /* IE9 Compat Modes */
  src: local('Ubuntu Light'), local('Ubuntu-Light'),
       url('../../fonts/ubuntu-v9-latin-300.eot?#iefix') format('embedded-opentype'), /* IE6-IE8 */
       url('../../fonts/ubuntu-v9-latin-300.woff2') format('woff2'), /* Super Modern Browsers */
       url('../../fonts/ubuntu-v9-latin-300.woff') format('woff'), /* Modern Browsers */
       url('../../fonts/ubuntu-v9-latin-300.ttf') format('truetype'), /* Safari, Android, iOS */
       url('../../fonts/ubuntu-v9-latin-300.svg#Ubuntu') format('svg'); /* Legacy iOS */
}
```

**Comment by SilkAndSlug** [comment]
Same here in `v1.0.4`:

    @font-face {
        font-family: 'Insula';
        src: url('../common/INSULA__.ttf?#iefix') format('truetype');
    }

    @font-face declaration doesn't follow the fontspring bulletproof syntax.
    src:        url('../common/INSULA__.ttf?#iefix') format('truetype');
